lib/phoenix_live_view/test/client_proxy.ex

```diff
@@ -356,6 +356,18 @@ defmodule Phoenix.LiveViewTest.ClientProxy do
     {:noreply, state}
   end
 
+  def handle_call({:open_browser, open_browser_fun, html}, from, state) do
+    filename = "#{state.root_view.topic}_#{Phoenix.LiveView.Utils.random_id()}"
+    html = DOM.maybe_wrap_html_element(html, state.root_view.endpoint.static_path("/priv/static/css/app.css"))
```

Comment:
I don't think we should hardcode app.css as you can be using any path. When we first start the `client_proxy`, we do have access to the whole HTML document. Maybe we should traverse the `head` at that moment and collect all `link` elements?

Reply:
Instead of traversing `head` to obtain all `link` elements, I decided to inject the whole head from `root_html` (https://github.com/phoenixframework/phoenix_live_view/pull/1181/commits/cfc8c1e3d9ecd77e170e35ef9e32fc5268ea751b#diff-39d73afd0f8def95e8eb610b5fa70619ff5a6755f2445aeea3d74d82a30f46a2R1113-R1121) so that covers link, style, and also other metadata like title, etc that could be useful. Sounds good?

Reply:
Sounds good but we may need to remove script tags though. Otherwise I am afraid the page will establish a LV connection to the server and run scripts that wipe out what is on the page.

Reply:
Good point, I'll do that.